### ISSUE TITLE
Please add support for Android FCM client

### ISSUE THREAD

**Issue by matsumura-f** [laravel-notification-channels/pusher-push-notifications#31]
https://github.com/laravel-notification-channels/pusher-push-notifications/blob/af6a44b91900b32a47fe4d34a5e32746c46175b5/src/PusherMessage.php#L297

I'd like to pushing to my Android FCM client 
example use [pusher/pusher-php-server]
```
// Install the server library with:
//   composer require pusher/pusher-php-server

require __DIR__ . '/vendor/autoload.php';

$pusher = new Pusher\Pusher('APP_KEY', 'APP_SECRET', 'APP_ID', array('cluster' => 'APP_CLUSTER'));

$pusher->notify(
  array("donuts"),
  array(
    'fcm' => array(
      'notification' => array(
        'title' => 'hello world',
        'icon' => 'androidlogo'
      ),
    ),
  )
);
```

**Comment by jessarcher** [comment]
@matsumura-f Can you confirm whether #42 resolves this? I've switched to the new Pusher Beams SDK, which uses FCM rather than GCM.

**Comment by atymic** [comment]
Fixed in #42 :)
